plugins/size/src/command.ts

```diff
@@ -78,7 +78,12 @@ const command: CliCommand = {
       description:
         'Comment on the Pull request with the results. (Only from CI + must set env var GH_TOKEN. In jenkins OWNER and REPO must also be set. In enterprise you must also set GITHUB_URL)',
       config: true
-    }
+    },
+    {
+      name: 'failureThreshold',
+      type: Number,
+      description: 'Failure Threshold for Size'
```

Comment:
Can you add `config: true`? This option seems like it should be configurable with the config file